android/framework/encode/CMakeLists.txt

```diff
@@ -52,6 +54,7 @@ target_sources(gfxrecon_encode
 target_include_directories(gfxrecon_encode
                            PUBLIC
                                ${CMAKE_BINARY_DIR}
-                               ${GFXRECON_SOURCE_DIR}/framework)
+                               ${GFXRECON_SOURCE_DIR}/framework
+                               ${CMAKE_CURRENT_LIST_DIR}/../../../../perfetto/sdk)
```

Comment:
What does this relative path translate into?  I changed this to:

/home/ianelliott/android/main1/vendor/arm/mali/valhall/cinstr/production/third_party/perfetto/vendor/sdk

Reply:
Looking at my Sokatoa source tree, I see that it includes perfetto and GFXR in (I use "$(TOP)" to indicate the top of the Sokatoa source tree):

- $(TOP)/external/perfetto/sdk
- $(TOP)/external/gfxreconstruct

Therefore, the above path (from $(TOP)/external/gfxreconstruct/android/framework/encode/CMakeLists.txt)
would translate to $(TOP)/external/perfetto/sdk.  True?